sigma-signature-library/sigma_signature/parser.py

```diff
@@ -56,9 +56,14 @@ class LogicTransformer(Transformer):
     def identifier_rule(self, args):
         # Call analyze on all rules in the rule directory to find matches for each event
         hits = analyze(event, str(rule), rules[rule])
```

Comment:
Its generally good practice to try and keep the return types of functions the same for different argument values. Its not a strict rule, but its a good habit for reducing potential errors.

In a case like this where you may return a list of hits, or a single hit, you might want to always return a list.

Reply:
done
https://github.com/CybercentreCanada/assemblyline-service-sigma/commit/01c717a8178278c061474c63fcffd6cabef98a08

Reply:
now in the code below this comment, you don't need that isinstance check

Reply:
was looking at the wrong commit there